### PR TITLE
Add .is-an.app.js File

### PR DESCRIPTION
**Pull request by Bhavesh-p-Patel**
### Link to Website

Link: 

---

- [ ] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/Bhavase-p-Patel.is-an.app.js
Comment (tarampampam) [inline domains/Bhavase-p-Patel.is-an.app.js:1]: Filename in lowercase, please

File: domains/Bhavase-p-Patel.is-an.app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+addSubDomain({
+    description: 'I want to publish my profile',
+    domain: 'is-an.app',
+    subdomain: 'Bhavesh-p-Patel',
```
Comment (tarampampam) [inline domains/Bhavase-p-Patel.is-an.app.js:4]: In lowercase, please

File: domains/Bhavase-p-Patel.is-an.app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    owner: {
+      repo: 'https://github.com/Bhavesh-p-Patel/Bhavesh-p-Patel.github.io',
+      email: 'bhavesh.patel@truestaz.com',
+    },
+    record: {
+      CNAME: 'Bhavesh-p-Patel.github.io',
```
Comment (tarampampam) [inline domains/Bhavase-p-Patel.is-an.app.js:10]: In lowercase, please

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Comment by tarampampam** [comment]
@Bhavesh-p-Patel How many domains do you plan to register else?

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
Closed due inactivity

**Comment by soni-ashish** [comment]
Give me proper reason to fix it @tarampampam

**Comment by tarampampam** [comment]
I mean if your company wants to publicize its developer profiles, why not do that using the company domain name? Now it looks like a mass subdomain registration, that takes a lot of my time and interferes with my offline work.

That's why I asked - "How many sub-domains do you plan to register?". Without the answer and explanation, I will not register one more.

**Pull request closed without merging**